pkg/api/server/register_containers.go

```diff
@@ -1176,15 +1176,19 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//    description: the name or ID of the container
 	//  - in: query
 	//    name: condition
-	//    type: string
-	//    description: |
-	//      wait until container is to a given condition. default is stopped. valid conditions are:
-	//        - configured
-	//        - created
-	//        - exited
-	//        - paused
-	//        - running
-	//        - stopped
+	//    type: array
+	//    items:
+	//      type: string
```

Comment:
I wish I could use named type as `items:` using `$ref:` but for `query` param it seems not to pass validation, although generators happily generate code :disappointed: